src/components/flex-container/examples/flex-container.tsx

```diff
@@ -90,31 +90,34 @@ export class FlexContainerExample {
 
     public render() {
         return [
-            <limel-flex-container justify="space-between">
```

Comment:
🤣 I don't know if we should remove this example all together, or keep it still…

Reply:
Well, we certainly shouldn't make _this_ change at least 🤣

But I guess `limel-flex-container` is deprecated, right? Then we can remove this example. We can make the docs for it private too.

Reply:
Yes, `limel-flex-container` is deprecated, and it seems we've already made the docs for it private. So absolutely, feel free to just remove this example 👍

Reply:
But not sure if someone who still uses it comes and wants to see or test or fiddle around with it... maybe it should be there, until we actually just get rid of it totally? Maybe I'm too stupid. If we don't want people to use it, perhaps examples should just be removed.

Reply:
Ok, I did it as the last commit

Reply:
People can always go an look at the docs for older versions 🙂